### PR TITLE
Created js file for enabling tech preview for console, updated master…

### PR DESCRIPTION
**Pull request by ewolinetz**
…-config for pod presets and console tech preview

Should address https://bugzilla.redhat.com/show_bug.cgi?id=1469986

### CODE REVIEW

File: roles/openshift_service_catalog/tasks/wire_aggregator.yml [hunk 1]
```diff
@@ -138,6 +143,10 @@
       value: [X-Remote-Group]
     - key: authConfig.requestHeader.extraHeaderPrefixes
       value: [X-Remote-Extra-]
+    - key: assetConfig.extensionScripts
+      value: [/etc/origin/master/openshift-ansible-catalog-console.js]
+    - key: kubernetesMasterConfig.apiServerArguments.runtime-config
+      value: [apis/settings.k8s.io/v1alpha1=true]
```
Comment (sdodson) [inline roles/openshift_service_catalog/tasks/wire_aggregator.yml]: Does this [] notation ensure that it's only an additive thing or would this overwrite whatever they have now? If it's overwriting I'm fine with this for now but we'll want to make sure it's additive as a followup.

Comment (ewolinetz) [inline roles/openshift_service_catalog/tasks/wire_aggregator.yml]: its to ensure that its a list, if it isn't added as a list item the master will fail to start up

Comment (ewolinetz) [inline roles/openshift_service_catalog/tasks/wire_aggregator.yml]: Im not entirely sure, but i believe it will overwrite currently

Comment (sdodson) [inline roles/openshift_service_catalog/tasks/wire_aggregator.yml]: @kwoodson is there an easy way to ensure yedit is additive only?

**Comment by spadgett** [comment]
openshift-ansible-catalog-console.js LGTM

**Pull request merged**